test/unit/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepositorySpec.scala

```diff
@@ -54,7 +54,7 @@ class CaseRepositorySpec extends BaseMongoIndexSpec
   private val repository = createMongoRepo
 
   private def createMongoRepo = {
-    new CaseMongoRepository(mongoDbProvider, new SearchMapper)
+    new CaseMongoRepository(mongoDbProvider, new SearchMapper) // TODO: we should mock `searchMapper`
```

Comment:
do you agree on this comment?

Reply:
I think this case we need to test it *with* SearchMapper. Otherwise the mapper could generate Json that doesn’t work with Mongo and we’d never know